shopgate/pwa
# Added automatic changelog creation for PWA v5.X and modified Makefile for stable releases

# Description

Please include a summary of the change. Please also include relevant motivation and context.

## Type of change


## How to test it

Create an rc release and check if if worked as it should and if the `CHANGELOG.md` files are created properly.
